scripts/taiseilib/version.py

```diff
@@ -97,3 +100,7 @@ def main(args):
 
     args = parser.parse_args(args[1:])
     print(get(args=args).format(template=args.format))
+
+if __name__ == '__main__':
+    import sys
+    main(sys.argv)
```

Comment:
You're actually expected to call `scripts/version.py` instead of this module directly.